apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -70,16 +75,22 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
             }
         } else if (allArguments[0] instanceof CommandData) {
             CommandData commandData = (CommandData) allArguments[0];
-            String command = commandData.getCommand().getName();
-            operationName = operationName + command;
-            addCommandData(dbStatement, commandData);
+            methodName = commandData.getCommand().getName();
+            operationName = operationName + methodName;
+            arguments = commandData.getParams();
         }
 
         AbstractSpan span = ContextManager.createExitSpan(operationName, peer);
         span.setComponent(ComponentsDefine.REDISSON);
-        Tags.DB_TYPE.set(span, "Redis");
+        Tags.CACHE_TYPE.set(span, "Redis");
         Tags.DB_INSTANCE.set(span, dbInstance);
```

Comment:
Please remove DB tags, we just need Cache tags.

Reply:
There is also an 'Tags.DB_STATEMENT' in batch processing. How to handle it? Change it to 'Tags.CACHE_ CMD'?

Reply:
I think `Tags.DB_STATEMENT` could be removed,  `DB_STATEMENT= cache.cmd + cache.key`. Right? 
For `DB_INSTANCE`, how about we add a tag `CACHE_INSTANCE` to indicate which cache node is requested?

Reply:
I agree with you.